installer/paymetheus/GNUmakefile

```diff
@@ -18,6 +19,12 @@ PROJFILES64+=Paymetheus/bin/x64/Release/*.dll
 PROJFILES64+=Paymetheus/bin/x64/Release/nativelibs
 PROJWALLET=github.com/decred/dcrwallet
 
+# nuget directories
+NUGETPKG=Paymetheus
+NUGETPKG+=Paymetheus.Decred
+NUGETPKG+=Paymetheus.Rpc
+NUGETPKG+=Paymetheus.Tests
```

Comment:
actually this is fine, so long as we don't end up building and/or distributing the test assemblies to users.